packages/deployer/terraform/airnode/modules/function/main.tf

```diff
@@ -0,0 +1,84 @@
+resource "random_uuid" "uuid" {
+}
+
+resource "null_resource" "fetch_lambda_files" {
+  provisioner "local-exec" {
+    command = <<EOC
+rm -rf ${local.tmp_dir}
+mkdir -p "${local.tmp_source_dir}" "${local.tmp_configuration_dir}"
+cp "${var.source_file}" "${local.tmp_source_dir}"
+cp "${var.configuration_file}" "${local.tmp_configuration_dir}"
+EOC
+  }
+
+  triggers = {
+    source_file_hash        = fileexists("${local.tmp_source_dir}/${var.source_file}") ? filesha256(var.source_file) : ""
+    configuration_file_hash = fileexists("${local.tmp_configuration_dir}/${var.configuration_file}") ? filesha256(var.configuration_file) : ""
+  }
+}
+
+resource "aws_iam_role" "lambda_role" {
+  name               = "${local.name}-role"
+  assume_role_policy = data.aws_iam_policy_document.role_policy.json
+}
+
+resource "aws_cloudwatch_log_group" "cloudwatch_log_group" {
+  name              = "/aws/lambda/${local.name}"
+  retention_in_days = 14
+}
+
+resource "aws_iam_role_policy" "lambda_log_role_policy" {
+  name   = "${local.name}-log-policy"
+  role   = aws_iam_role.lambda_role.id
+  policy = data.aws_iam_policy_document.cloudwatch_log_policy.json
+}
+
+resource "aws_iam_role_policy" "invoke_lambda_role_policy" {
+  count = length(var.invoke_targets) != 0 ? 1 : 0
+
+  name   = "${local.name}-invoke-policy"
+  role   = aws_iam_role.lambda_role.id
+  policy = data.aws_iam_policy_document.lambda_invoke_policy.json
+}
+
+resource "aws_lambda_function" "lambda" {
+  filename                       = "${local.tmp_output_dir}/${local.name}.zip"
+  source_code_hash               = data.archive_file.lambda_zip.output_base64sha256
+  function_name                  = local.name
+  handler                        = var.handler
+  runtime                        = "nodejs14.x"
```

Comment:
Can we make this constant a variable so it stands out? 

How will this work when we plan to support custom airnodes? 
1) Will we compile them from the source code of the user?
2) Will we use their compiled code?
If we use 2) we need to make it clear we support node14 and people should compile to that version.

Actually, we should make that clear anyway, because some features (that the custom airnode uses) might not be available in node14.

Reply:
I'd say we're aiming for option 2 and this will most likely become configurable as well. For now, I would focus on our airnode implementation(s) and our deployment process and once that's done start providing options for customization.